src/lib/openapi/spec/health-overview-schema.ts

```diff
@@ -40,29 +47,44 @@ export const healthOverviewSchema = {
         },
         members: {
             type: 'number',
```

Comment:
My guess is this is a non-zero natural number, right? A project **must** have an owner?
```suggestion
            type: 'integer',
            minimum: 1,
```

Reply:
I suspect you are correct, but I'll have a look

Reply:
It defaults to being 0 - so it's not nullable, I am unsure if it's possible to completely abandon a project?

Reply:
Got it. In that case, I'd go for `type: integer, minimum: 0`, I think.